tfhe/docs/getting_started/benchmarks.md

```diff
@@ -29,13 +29,42 @@ This measures the execution time of a single binary Boolean gate.
 
 
+## Integer
+This measures the execution time for some operation sets of tfhe-rs::integer.
+
+| Operations\bit_size | `FheUint8` | `FheUint16` | `FheUint32` | `FheUint40` | `FheUint64` | `FheUint128` | `FheUint256` |
+|---------------------|------------|-------------|-------------|-------------|-------------|--------------|--------------|
+| add                 | 153 ms     | 198 ms      | 246 ms      | 293 ms      | 308 ms      | 426 ms       | 483 ms       |
+| sub                 | 153 ms     | 197 ms      | 245 ms      | 289 ms      | 312 ms      | 435 ms       | 515 ms       |
+| bitand              | 37.5 ms    | 40.3 ms     | 42.8 ms     | 43.2 ms     | 46.3 ms     | 63.0 ms      | 63.4 ms      |
+| bitor               | 37.4 ms    | 39.9 ms     | 41.1 ms     | 42.1 ms     | 45.4 ms     | 56.0 ms      | 58.1 ms      |
+| bitxor              | 36.5 ms    | 40.4 ms     | 41.7 ms     | 42.6 ms     | 45.3 ms     | 56.5 ms      | 59.6 ms      |
+| negation            | 142 ms     | 195 ms      | 242 ms      | 282 ms      | 297 ms      | 419 ms       | 498 ms       |
+| not_equal           | 81.0 ms    | 81.1 ms     | 121 ms      | 123 ms      | 126 ms      | 134 ms       | 135 ms       |
+| equal               | 80.4 ms    | 81.2 ms     | 117 ms      | 120 ms      | 124 ms      | 134 ms       | 136 ms       |
+| greater_or_equal    | 107 ms     | 145 ms      | 191 ms      | 229 ms      | 246 ms      | 296 ms       | 369 ms       |
+| greater_than        | 110 ms     | 146 ms      | 191 ms      | 231 ms      | 245 ms      | 294 ms       | 365 ms       |
+| left_shift          | 193 ms     | 237 ms      | 297 ms      | 380 ms      | 445 ms      | 548 ms       | 1.07 s       |
+| right_shift         | 193 ms     | 237 ms      | 305 ms      | 380 ms      | 435 ms      | 551 ms       | 1.04 s       |
+| less_or_equal       | 103 ms     | 144 ms      | 195 ms      | 229 ms      | 248 ms      | 295 ms       | 368 ms       |
+| less_than           | 103 ms     | 147 ms      | 188 ms      | 226 ms      | 240 ms      | 292 ms       | 360 ms       |
+| max                 | 184 ms     | 226 ms      | 280 ms      | 322 ms      | 340 ms      | 394 ms       | 510 ms       |
+| min                 | 188 ms     | 226 ms      | 280 ms      | 326 ms      | 339 ms      | 399 ms       | 516 ms       |
+| mul                 | 277 ms     | 385 ms      | 545 ms      | 763 ms      | 1.16 s      | 3.10 s       | 10.9 s       |
+| rotate_left         | 195 ms     | 241 ms      | 300 ms      | 380 ms      | 428 ms      | 530 ms       | 1.05 s       |
+| rotate_right        | 198 ms     | 243 ms      | 299 ms      | 379 ms      | 427 ms      | 541 ms       | 1.08 s       |
+
+All timings are related to parallelized Radix-based integer operations, where each block is encrypted using the default parameters (i.e., PARAM\_MESSAGE\_2\_CARRY\_2, more information about parameters can be found [here](../fine_grain_api/shortint/parameters.md)). 
```

Comment:
missing the KS_PBS suffix, though the alias exists so not sure which is best to advertise